modules/agrvate/main.nf

```diff
@@ -0,0 +1,38 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process AGRVATE {
+    tag "$meta.id"
+    label 'process_low'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::agrvate=1.0" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/agrvate:1.0--hdfd78af_0"
+    } else {
+        container "quay.io/biocontainers/agrvate:1.0--hdfd78af_0"
+    }
+
+    input:
+    tuple val(meta), path(fasta)
```

Comment:
I haven't specified a separate `path` input for the `agrvate_databases` since as per the [manual](https://github.com/VishnuRaghuram94/AgrVATE#usage
) 


```
-d   Path to agrvate_databases (Not required if installed using Conda) [alternate: --databases]
```